app/views/citizens/property_values/show.html.erb

```diff
@@ -11,8 +14,14 @@
           input: :property_value
         ) do %>
 
-      <%= govuk_fieldset_header t('.field_set_header'), padding_below: 6 %>
-      <%= form.govuk_text_field :property_value, input_prefix: t('currency.gbp'), class: 'govuk-input govuk-!-width-one-third' %>
+      <%= govuk_fieldset_header content_for(:page_title), padding_below: 6 %>
+      <%= form.label :property_value, t('.label'), class: 'govuk-label' %>
+      <%= govuk_hint t('.hint'), id: :property_value_hint %>
+      <%= govuk_error_message form.object.errors[:property_value].first %>
+      <%= govuk_currency_input_wrapper do %>
+        <% input_error_class = govuk_error_class(form.object.errors[:property_value]) %>
+        <%= form.text_field :property_value, class: "govuk-input govuk-!-width-one-third govuk-currency-input__inner__input #{input_error_class}" %>
+      <% end %>
```

Comment:
Lines 18-24 look to have brought in the old way of defining the text field. I think the old line 15 was the correct code. So looks like a merge error.